windtools/plotting.py

```diff
@@ -1234,6 +1233,52 @@ def plot_spectrum(datasets,
 #
 # ---------------------------------------------
 
+def addLabels(axs,loc='upper right', fontsize=14, alpha=0.6, pad=6, start='a', **bbox_props):
```

Comment:
Everything mostly looks good. I added a way to customize the label bounding box further (thinking ahead to potential use cases for myself). I like having the subplot labels on the inside (rather than outside, which what the orig code did) — but I'm not crazy about the default position being in the upper right. And I'm wondering if there's a way we can preserve backwards compatibility... Let me propose some additional changes...

Reply:
sounds good. The way I have it right now generated the label on the inside, not on the outside.